tree/ntuple/v7/src/RColumnElement.cxx

```diff
@@ -180,6 +180,12 @@ ROOT::Experimental::Internal::GenerateColumnElement(EColumnCppType cppType, ECol
    return nullptr;
 }
 
+std::unique_ptr<ROOT::Experimental::Internal::RColumnElementBase>
+ROOT::Experimental::Internal::GenerateColumnElement(const RColumnElementBase::RIdentifier &elementId)
+{
+   return GenerateColumnElement(elementId.fInMemoryType, elementId.fOnDiskType);
```

Comment:
I believe this won't compile yet in this commit because this `GenerateColumnElement` doesn't exist yet... Maybe swap order with the next change?

Reply:
Thanks! I added another commit for this overload.